src/server/game/AI/SmartScripts/SmartScript.cpp

```diff
@@ -297,7 +297,11 @@ void SmartScript::ProcessAction(SmartScriptHolder& e, Unit* unit, uint32 var0, u
             {
                 if (IsUnit(target))
                 {
-                    target->PlayDirectSound(e.action.sound.sound, e.action.sound.onlySelf ? target->ToPlayer() : nullptr, e.action.sound.keyBroadcastTextId);
+                    if (e.action.sound.distance == 1)
+                        target->PlayDistanceSound(e.action.sound.sound, e.action.sound.onlySelf ? target->ToPlayer() : nullptr, e.action.sound.keyBroadcastTextId);
```

Comment:
`PlayDistanceSound` doesn't have `keyBroadcastTextId` arg (no such field in that packet)

Reply:
That argument seems to be a deviation from the originating (cherry-picked) commit https://github.com/TrinityCore/TrinityCore/commit/37995ade2991ed27845728eb6a6dd95f3a490849 , it is not present there.

Reply:
Will fix that